moviebuffs-ui-react/src/components/NavBar.js

```
@@ -1,92 +1,65 @@
+    return cart
+      .map(item => item.quantity)
+      .reduce((accumulator, currentValue) => accumulator + currentValue, 0);
+  };
+
+  const logoutHandler = () => {
+    cleanState();
+    navigate("/");
+    window.location.reload(); // Force reload to clear any remaining state
+  };
+
+  const authenticatedLinks = user && user.access_token ? (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)